cmd/usecasevalidator/main.go

```diff
@@ -71,7 +71,12 @@ func (f cliFlags) isUpdateNeeded() bool {
 	return f.analytics || f.creator || f.removeRelations || f.exclude || f.rules != ""
 }
 
-const anytypeProfileFilename = addr.AnytypeProfileId + ".pb"
+const (
+	anytypeProfileFilename = addr.AnytypeProfileId + ".pb"
+
+	filesPrefix       = "files"
```

Comment:
We have the same constants in export package, is is possible to reuse it somehow?

Reply:
Oh yeah, did not think of them first...
Fixed!